addons/website/views/snippets/s_product_list.xml

```diff
@@ -3,62 +3,94 @@
 
 <template name="Items" id="s_product_list">
     <t t-set="url" value="url or '#'"/>
-    <section class="s_product_list">
-        <div class="container-fluid">
+    <section class="s_product_list pt64 pb64" data-vcss="001">
+        <div class="container">
+            <h3 class="mb-4">Our finest selection</h3>
```

Comment:
Doing that will prevent the customization made in the WB, you should remove it :) 
<img width="1454" alt="Capture d’écran 2024-08-01 à 16 39 16" src="https://github.com/user-attachments/assets/1e6f5a3c-76d3-4aa7-88ee-0bb281a62fa5">